src/server/pachyderm_test.go

```diff
@@ -2644,6 +2644,54 @@ func TestPutFileURL(t *testing.T) {
 	require.True(t, fileInfo.SizeBytes > 0)
 }
 
+func TestArchiveAllWithPipelines(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	t.Parallel()
+	c := getUsablePachClient(t)
+	dataRepo := uniqueString("TestUpdatePipeline_data")
+	require.NoError(t, c.CreateRepo(dataRepo))
+	numPipelines := 10
+	var outputRepos []*pfsclient.Repo
+	for i := 0; i < numPipelines; i++ {
+		pipelineName := uniqueString("pipeline")
+		outputRepos = append(outputRepos, &pfsclient.Repo{Name: pipelineName})
+		require.NoError(t, c.CreatePipeline(
+			pipelineName,
+			"",
+			[]string{"cp", path.Join("/pfs", dataRepo, "file1"), "/pfs/out/file"},
+			nil,
+			1,
+			[]*ppsclient.PipelineInput{{Repo: client.NewRepo(dataRepo)}},
+			false,
+		))
+	}
+	commit, err := c.StartCommit(dataRepo, "", "")
+	require.NoError(t, err)
+	_, err = c.PutFile(dataRepo, commit.ID, "file1", strings.NewReader("file1\n"))
+	_, err = c.PutFile(dataRepo, commit.ID, "file2", strings.NewReader("file2\n"))
+	_, err = c.PutFile(dataRepo, commit.ID, "file3", strings.NewReader("file3\n"))
+	require.NoError(t, err)
+	require.NoError(t, c.FinishCommit(dataRepo, commit.ID))
+	commitInfos, err := c.FlushCommit([]*pfsclient.Commit{commit}, nil)
+	require.NoError(t, err)
+	require.Equal(t, numPipelines, len(commitInfos))
+
+	require.NoError(t, c.ArchiveAll())
+	listCommitRequest := &pfsclient.ListCommitRequest{
+		Repo:       outputRepos,
+		CommitType: pfsclient.CommitType_COMMIT_TYPE_NONE,
+		Block:      false,
+	}
+	listCommitResponse, err := c.PfsAPIClient.ListCommit(
+		context.Background(),
+		listCommitRequest,
+	)
```

Comment:
I think you could just do this with `c.ListCommit` doesn't look like you're passing an arguments it doesn't support.